workshops.md

```diff
@@ -0,0 +1,333 @@
+annotation, abundance estimation, and data visualization. The second
+two (three) days will offer participants the opportunity to apply the
+topics covered during the first two days of the workshop to their own
+data with the support of other participants and the instructors.
+
+This workshop will not cover 16s data analysis.
+
+#### [Workshop registration](https://registration.genomecenter.ucdavis.edu/events/DIBSI_EM/)
+
+#### [Housing registration](https://registration.genomecenter.ucdavis.edu/events/Housing/)
+
+### Non-model RNAseq, bring your own data
+
+Dates: July 17-21
+
+Instructors: Tessa Pierce, Jane Khudyakov, Lisa Cohen
+
+Contact: [Lisa Cohen](mailto:ljcohen@ucdavis.edu)
+
+The focus of this hands-on tutorial will be RNAseq de novo assembly
+and quantification. It is intended for participants with Illumina
+poly(A) selected RNA sequencing data from a non-model organism with no
+closely-related reference genome who would like assistance analyzing
+and learning more about the software tools commonly used in this type
+of analysis. Time will be spent working on data brought by attendees
+with the idea to get through all steps of a typical pipeline
+workflow. We will provide scripts, example sets of data to work with,
+and cloud computing resources. Attendees should already have
+some familiarity with using command line software tools and
+beginning-level next-generation sequence analysis materials (see
+http://ivory.idyll.org/dibsi/ANGUS.html). This workshop is ideal for
+alumni of previous years of the ANGUS workshop at MSU Kellogg
+Biological Station or attendees from this year's 2-week workshop at UC
+Davis. We will provide basic training in the use of GitHub for
+collaborative work. If you would like assistance adapting our
+materials to run on the computing resources at your home institution,
+please let us know.
+
+For example materials, see http://eel-pond.readthedocs.io/en/latest/  
+
+#### [Workshop registration](https://registration.genomecenter.ucdavis.edu/events/DIBSI_RNASeq_byod/)
+
+#### [Housing registration](https://registration.genomecenter.ucdavis.edu/events/Housing/)
+
+### Introduction to R
+
+Dates: July 17-21
+
+Instructor: [Michael Koontz](mailto:mikoontz@gmail.com)
+
+Join us for an interactive, week-long introduction to the programming
+language R!
+
+R is a powerful, cross-platform, open-source, and free software that
+has been widely adopted across a number of science fields. While
+incredibly useful, it can also be daunting to learn. This course
+doesn't require any prior programming experience. We'll teach you the
+basics of R by writing code together and setting up our computers the
+same way you will to work on your own data after the workshop. By the
+end of the week, you'll be able to input, organize, and summarize data
+in R. You'll also learn how to visualize and present data using
+publication-quality plots and dynamic documents that combine
+descriptive writing with the results of your code.
+
```

Comment:
The course will focus on laying a groundwork of basic R skills to enable future self-teaching of specific use cases. However, enrollees are encouraged to reach out to the instructor [Michael Koontz](mailto:mikoontz@gmail.com) if there are particular topics that they think would be especially valuable to cover, and we’ll try to work them into the curriculum.